lib/bundle/dumper.rb

```diff
@@ -20,6 +20,7 @@ def self.dump_brewfile
       content << cask_before_formula
       content << BrewDumper.dump
       content << cask_after_formula
+      content << MacAppStoreDumper.dump
```

Comment:
I think it's enough to assume that if you don't have `mas` installed you don't want this behaviour, right?